keyboards/atreus/keymaps/ibnuda/rules.mk

```
@@ -0,0 +1,9 @@
+# Build Options
+#   change yes to no to disable
+#
+
+MOUSEKEY_ENABLE  = yes     # Mouse keys(+4700)
+COMBO_ENABLE	 = yes
+TAP_DANCE_ENABLE = yes
+
+DEFAULT_FOLDER = atreus/greasy-promicro
```

Shouldn't this just be

```suggestion
DEFAULT_FOLDER = atreus/promicro
```

?